clippy_lints/src/manual_strip.rs

```diff
@@ -1,21 +1,24 @@
 use crate::consts::{constant, Constant};
 use crate::utils::usage::mutated_variables;
 use crate::utils::{
-    eq_expr_value, higher, match_def_path, multispan_sugg, paths, qpath_res, snippet, span_lint_and_then,
+    eq_expr_value, higher, match_def_path, multispan_sugg, paths, qpath_res, snippet, span_lint_and_then, LimitStack,
```

Comment:
```suggestion
    eq_expr_value, higher, match_def_path, multispan_sugg, paths, qpath_res,
    snippet, span_lint_and_then, LimitStack,
```
Helps reduce occurrence of long lines